internal/controllers/machine/drain/drain.go

```diff
@@ -448,46 +448,38 @@ func (r EvictionResult) ConditionMessage(nodeDrainStartTime *metav1.Time) string
 
 	conditionMessage := fmt.Sprintf("Drain not completed yet (started at %s):", nodeDrainStartTime.Format(time.RFC3339))
 	if len(r.PodsDeletionTimestampSet) > 0 {
-		conditionMessage = fmt.Sprintf("%s\n* Pods with deletionTimestamp that still exist: %s",
+		conditionMessage = fmt.Sprintf("%s\n* Pod %s with deletionTimestamp and still not removed from the node",
 			conditionMessage, PodListToString(r.PodsDeletionTimestampSet, 3))
 	}
 	if len(r.PodsFailedEviction) > 0 {
 		sortedFailureMessages := maps.Keys(r.PodsFailedEviction)
 		sort.Strings(sortedFailureMessages)
 
-		conditionMessage = fmt.Sprintf("%s\n* Pods with eviction failed:", conditionMessage)
```

Comment:
Wondering if we are now losing the context info that we triggered eviction for these Pods

Reply:
If we want to make sure we don't have any sort of additional prefix we can string match on the error and treat PDB differently. So we would end up with e.g.
```yaml
* Pod pod-5-to-trigger-eviction-pdb-violated-1: cannot evict pod as it would violate the pod's disruption budget. The disruption budget pod-5-pdb needs 20 healthy pods and has 20 currently
* Pod pod-6-to-trigger-eviction-some-other-error: failed to evict Pod: some other error 1
```
(including lower case for "Cannot")